model/ftn/ww3_prnc.ftn

```diff
@@ -1823,8 +1854,8 @@
 !
                 IF (NFCOMP.EQ.2) THEN
 !/O3             IF ( IAPROC .EQ. NAPOUT )    WRITE (NDSO,976) ' (2) '
-                    CALL INTERP(YC, JX21, JX22, JY21, JY22, XD11, XD12,&
-                                             XD21, XD22, FILLVALUE, FA)
+                    CALL INTERP(MXM, MYM, YC, JX21, JX22, JY21, JY22,  &
```

Comment:
Is this a separate bug fix?
It looks like the call to interp was missing the dimension variables MXM and MYM.

Reply:
yes, it was crashing when compiling so I've corrected it

Reply:
Is there an issue associated with this fix?

Reply:
I've just created an issue #402 to document it